### PR TITLE
Merge 2.x docs versions

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
- Fixes https://github.com/RasaHQ/rasa-sdk/issues/622

**How to review this huge PR**
Sorry about the big diff, I didn't have the time to create separate PRs. The changes are pretty minimal, you can focus your review on the commits that merge docs (instead of the ones that remove old versions of the docs):
- ["merge 2.2.0 docs into 2.0.0"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/7244ea9)
- ["rename docs for 2.0.0 to 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/ef538a7)
- ["merge 2.3.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/d4da453)
- ["merge 2.4.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/560d214)
- ["merge 2.5.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/95ecf19)
- ["merge 2.6.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/a5b71a2)
- ["merge 2.7.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/a022108)
- ["merge 2.8.0 docs into 2.x"](https://github.com/RasaHQ/rasa-sdk/pull/626/commits/3c16b75)

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

File: .github/workflows/continous-integration.yml [hunk 1]
```diff
@@ -1,198 +0,0 @@
-name: Continous Integration
```
Comment (m-vdb) [inline .github/workflows/continous-integration.yml:1]: This workflow was renamed on `main` (there was a typo in the name) but the `documentation` branch wasn't cleaned up.

**Review by melindaloubser1 — CHANGES_REQUESTED** [review]
Thanks for the links, made that way  more review-able!
Just one thing: This file https://github.com/RasaHQ/rasa-sdk/pull/626/files#diff-62c30ba09428f9c2115eab0aac2ad7ea250d1a22efdf60ec6471ff9b469ea10e
Should have a note that the `--log-file` argument was only introduced in 2.3.0.

Other than that all good!

**Comment by m-vdb** [comment]
fixed with 28f741a

**Review by melindaloubser1 — APPROVED** [review]
🚀

**Pull request merged**